README.md

```diff
@@ -1,33 +1,32 @@
-# Project
+# Azure Terraform Verified Modules
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+> This repo contains the source code of all currently available Terraform modules verified by Microsoft Azure. All verified modules will meet the quality pipeline established by Microsoft Azure team, which guarantee their high quality.    
```

Comment:
This is actually not true. This repo contains the documentation for verified modules and pointers to the module repos. 

Also please remove the guarantee ", which guarantee their high quality" at the end. That is too strong of a statement.